File: services/utils-service/src/main/java/com/sgpttt/UtilsService/repository/ProtocolSearchRepository.java
package com.sgpttt.UtilsService.repository;

import com.sgpttt.UtilsService.entity.ProtocolDocument;
import org.springframework.data.elasticsearch.repository.ElasticsearchRepository;

public interface ProtocolSearchRepository extends ElasticsearchRepository<ProtocolDocument, Long> {
    
}

File: services/protocols-service/src/main/kotlin/com/sgptt/protocolsservice/repository/ProtocolSearchRepository.kt
package com.sgptt.protocolsservice.repository

import com.sgptt.protocolsservice.repository.entity.ProtocolDocument
import org.springframework.data.elasticsearch.repository.ElasticsearchRepository
import org.springframework.stereotype.Repository

@Repository
interface ProtocolSearchRepository : ElasticsearchRepository<ProtocolDocument, Long>
